deploy/overlays/kustomization.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC
```

Comment:
Was this meant to be under deploy/overlays/buoyant-country-377016/kustomization.yaml?

Reply:
This should not be checked in. Removed it.